src/app/business/routes/objective/sagas/index.js

```diff
@@ -75,14 +58,7 @@ function* fetchItem({payload}) {
 
 function* fetchPersistent(request) {
     const state = yield select();
-    let jwt;
-
-    if (typeof window !== 'undefined') {
-        const cookies = cookie.parse(window.document.cookie);
-        if (cookies['header.payload']) {
-            jwt = cookies['header.payload'];
-        }
-    }
+    const jwt = getJWTFromCookie();
```

Comment:
Same about the incomplete refacto. 3rd time, always about `fetchPersistent`, it looks like I'm really missing something!

Reply:
same yes